.github/workflows/ci_cron_daily.yml

```diff
@@ -42,6 +42,13 @@ jobs:
             python: 3.8
             toxenv: pyinstaller
 
+          - name: Python 3.10 with all optional dependencies and pre-releases
+            os: ubuntu-latest
+            python: '3.10'
+            toxenv: py310-test-alldeps-predeps
+            toxargs: -v
+            toxposargs: --remote-data=any
```

Comment:
Is this overkill? I can dial this down from 11 if it is.

Reply:
My sense would be not to do `remote-data` since you'll just get false failures due to URL timeouts, etc...

Reply:
This is cron so won't affect PR and therefore I think the more comprehensive the run the better.

Reply:
Also good point, but my worry is that someone has to spend time looking at every failure, and it would help to ensure that failures are real!

Though perhaps really what we need is a flag that all URL timeouts are ignored... And use that for all remote data runs except a single weekly (monthly?) one...

Reply:
I don't think it is possible to run remote data tests but ignore the failures. I either include them or don't.

Reply:
Yes, more wishful thinking, that almost always failures of the remote data tests are transient URL timeouts. Would be nice if we could ignore those! But an easier solution might be the preloaded cache that should be possible these days. Anyway, rather off-topic!

Reply:
I'd vote not to run remote-data, to minimize the number of false positives. But really *you* should get to decide, since it typically is you who notices something is amiss!

Reply:
I would think doing remote-data is more valuable than not for a cron job, after all, timeouts are not really expected (and if they are persistent, the URLs should be switched over). If some of the tests are flaky, then use pytest-rerunfailures instead of skipping running them.

But I agree that it should be Pey Lian who decides.

Reply:
Hmm the dev job in CI uses remote data

https://github.com/astropy/astropy/blob/f6c190f32b7fe07316ed51881efff2b75eaf458b/.github/workflows/ci_workflows.yml#L134

but not the weekly cron

https://github.com/astropy/astropy/blob/f6c190f32b7fe07316ed51881efff2b75eaf458b/.github/workflows/ci_cron_weekly.yml#L49

so I'll ponder a bit.